.devops/templates/runpublishvrscreenshot.yml

```diff
@@ -0,0 +1,47 @@
+parameters:
```

Comment:
nit: sorry I missed this earlier but can we rename this file as `run-publish-vr-screenshot.yml` for better readability?

Reply:
I'll do it next PR where we will add check for affected packages